specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/examples/DiskMigrations/Get.json

```diff
@@ -0,0 +1,52 @@
+{
+  "parameters": {
+    "subscriptionId": "04666444-56f0-4d4f-afc5-dbd6b134b084",
+    "location": "redmond",
+    "migrationId": "ec9f79cf-e4d7-4d37-926a-e69c2198be8a",
+    "api-version": "2018-07-30-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/04666444-56f0-4d4f-afc5-dbd6b134b084/providers/Microsoft.Compute.Admin/locations/redmond/diskmigrations/ec9f79cf-e4d7-4d37-926a-e69c2198be8a",
+        "name": "redmond/ec9f79cf-e4d7-4d37-926a-e69c2198be8a",
+        "type": "Microsoft.Compute.Admin/locations/diskmigrations",
+        "location": "redmond",
```

Comment:
no need to have "location" on proxy types

Reply:
I think proxy should have location. No proxy resource doesn't need the location.

Reply:
Can you clarify?

Reply:
For example, "publisher" resource of CRP. It is proxy only resource. And it has locations. 

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/Microsoft.Compute/stable/2017-03-30/compute.json

https://msazure.visualstudio.com/One/_git/AzureStack-Services-Compute?path=%2Fsrc%2FDeployment%2FNuget%2FRegistration%2FtenantTemplate.json&version=GBmaster

Reply:
Publisher shouldn't have a location and I recall having a conversation with Dwayne and he said it is there for legacy reasons so they can't remove it.